### PR TITLE
Claudio/fix wasm hash for

### PR DESCRIPTION
**Pull request by crusso**
@nomeata Look ok to you?

### CODE REVIEW

**Comment by osa1** [comment]
What is this PR about? Would be good to at least have a short PR description.

**Comment by crusso** [comment]
It's trying to fix #2643 which in turn is trying to fix the perf regression reporting

File: ci-pr.nix [hunk 1]
```diff
@@ -34,7 +34,7 @@ let
           fi
         done
 
-        if ! find -name \*.wasm | grep -q .
+        if ! ls */*.wasm
```
Comment (nomeata) [inline ci-pr.nix:37]: Hmm, I dislike using non- matching globs like this. And using LS should not be needed: the check is already “done” when the bash is done expanding the glob, the `ls` kinda does unnecessary extra stuff.

It's surprising that this is relatively non-trivial to do elegantly…

Comment (nomeata) [reply ci-pr.nix:37]: How about this? https://stackoverflow.com/a/34195247

Comment (crusso) [reply ci-pr.nix]: compgen isn't available in the CI env

Comment (crusso) [reply ci-pr.nix]: @nomeata this ok?

**Review by nomeata — APPROVED** [review]
This produces a large command line and may break once we have too many test files, but it breaks in the happy case (so it’s safe), and we can worry then.

**Pull request merged**